nifi-assembly/LICENSE

```diff
@@ -1729,4 +1729,20 @@ This product bundles 'jbzip2' which is available under an MIT license.
     AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
     LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-    THE SOFTWARE.
\ No newline at end of file
+    THE SOFTWARE.
```

Comment:
This whole license section can be removed.  This is the assembly license which is to cover all binary artifacts and source in the build of nifi itself.  The dependency of java-grok is binary only (not source) and is ASLv2 so nothing needs to be in this license for it.  There should be an entry for this in the notice similar to the many ASLv2 examples in there.   The only thing needing mentioned then is the copyright line from the project's license file https://github.com/thekrakken/java-grok/blob/master/LICENSE.  Also, this nifi-asembly/NOTICE change needed will also need to be in the NOTICE of the nifi-standard-nar as well.

Lots of words above but the short version is "No license change needed.  Just add a small section to the nar NOTICE and assembly NOTICE to reflect this ASLv2 dependency specifically because it has a copyright reference in the license."